src/task.rs

```diff
@@ -2126,6 +2204,15 @@ impl TaskMetrics {
         mean(self.total_fast_poll_duration, self.total_fast_poll_count)
     }
 
+    /// The average time taken for a task with a short scheduling delay to be executed after being
+    /// scheduled.
```

Comment:
Can you follow the documentation pattern of [`mean_fast_poll_duration`](https://docs.rs/tokio-metrics/0.2.0/tokio_metrics/struct.TaskMetrics.html#method.mean_fast_poll_duration) here?